Basic/ClientDriven/Assets/Scripts/ClientServerBaseNetworkBehaviour.cs

```diff
@@ -0,0 +1,35 @@
+namespace Unity.Netcode.Samples
+{
+    /// <summary>
+    /// Helper to reduce the amount of times you forget about isServer/isClient checks. With this, all classes are server driven by default,
+    /// which can be overriden with the virtual properties Both/ServerOnly/ClientOnly
+    /// This really is an experiment, there will still be Netcode events called on disabled GameObjects. This class might give a false sense of security
+    /// </summary>
+    public abstract class ClientServerBaseNetworkBehaviour : NetworkBehaviour
```

Comment:
I really don't know about that class. Doesn't it hide the fact that users will need to deal with IsServer or IsClient code in a real game?